### ISSUE TITLE
Installation doesn’t pull the most recent release

### ISSUE THREAD

**Issue by fbruetting** [flatpak/flatpak#2007]
## Linux distribution and version
Fedora 28

## Flatpak version
0.99.3

## Description of the problem
Today I installed Thunderbird via Flatpak. Directly afterwards I tried to run it, but that sadly failed. So I just out of curiosity ran `flatpak update` to see if some things might change. Then I was quite surprised by getting a new version of Thunderbird – about two minutes after installation.

So if this didn’t happen to be exactly that point in time, shortly before a new version was pushed onto the server, it looks like that either the repo doesn’t get synced at installation, or there may be some other strange thing happening here. Even when the repo just gets synced once a day or so, it would be cool if the attempt to download an older version (without exact specification) could get altered by the server to downloading the most recent version.

## Steps to reproduce
Don’t know if that’s possible…

**Comment by alexlarsson** [comment]
How did you install it? Just a regular "flatpak install"?

**Comment by fbruetting** [comment]
Yes.

**Comment by mwleeds** [comment]
This issue was created at 2018-08-23 1:32 PM PDT and there was a Thunderbird update at 2018-08-23 2:44 AM PDT, so it's possible that's the explanation, depending on how much time passed between when this happened and when you created the issue.

```
$ flatpak remote-info --log flathub org.mozilla.Thunderbird
Ref: app/org.mozilla.Thunderbird/x86_64/stable
ID: org.mozilla.Thunderbird
Arch: x86_64
Branch: stable
Collection ID: org.flathub.Stable
Date: 2018-08-23 09:44:20 +0000
Subject: [WIP] Revert "V60 fix" (#49) (dfb977eb)
Commit: 2557d76300a4265d5c64120815f34a141a6e5cbb5aaab0e457f75bd0f0a913f4
Parent: 56f777252f901346eaeb5ddebd56438b19965079866026aa93237d7589f3538d
Download size: 62.3 MB
Installed size: 135.6 MB
Runtime: org.gnome.Platform/x86_64/3.28
Sdk: org.gnome.Sdk/x86_64/3.28
History:
 Subject: Update icons (#48) (bfbd06f5)
 Date: 2018-08-10 10:50:28 +0000
 Commit: 56f777252f901346eaeb5ddebd56438b19965079866026aa93237d7589f3538d

 Subject: V60 fix (#45) (9a967e3a)
 Date: 2018-08-10 00:18:51 +0000
 Commit: 9eae6cc06268c40383cf75e42c37bc6faa83a4b06d834642eeb390c6fd88c3b2

 Subject: Revert "[Flatpak] Thunderbird's build system has changed" (311b84b4)
 Date: 2018-08-07 22:47:40 +0000
 Commit: 6d772dbdd5ca95dfa70121d1d1cb91c3f4a1501a330c24c5be36a24e16ac7471

 Subject: appdata: Use Linux screenshot (#42) (50cc4313)
 Date: 2018-07-17 09:30:42 +0000
 Commit: 109e698ef6ecc917b2a49168e0e782cac8b76f8e744bbd90c11b4db33a5dc676

```

**Comment by fbruetting** [comment]
Could be the case, is there a history command in flatpak?

**Comment by mwleeds** [comment]
yeah, it's `remote-info --log`

**Comment by fbruetting** [comment]
Ah, I meant for seeing when I updated it locally (like what `dnf history` does).

**Comment by mwleeds** [comment]
No, there's no transaction history command, although perhaps you could look at the file modification times with `ls -l /var/lib/flatpak/repo/refs/remotes/flathub/app/org.mozilla.Thunderbird/x86_64/`

**Comment by matthiasclasen** [comment]
Maybe we should talk about adding some logging
